### PR TITLE
Added support for Gnome[-Terminal

### PR DESCRIPTION
**Pull request by jamietanna**
Generate settings for gnome-terminal, and automagically import them when calling wp-change.

Used with gnome-terminal that is default on Ubuntu.

### CODE REVIEW

**Comment by jamietanna** [comment]
Update: This doesn't 100% work, currently working on fix.

File: py/color_detect.py [hunk 1]
```diff
@@ -55,8 +57,10 @@ def to_hsv(c):
     else:
         n = int(sys.argv[2])
 
-
     i = 0
+
+    gn = ""
```
Comment (everett1992) [inline py/color_detect.py:62]: What does this var do?

**Comment by everett1992** [comment]
Overall a good commit and I'll merge it. It might be good to add command line options to generate colors for different terminals instead of generating all color schemes.

File: py/color_detect.py (hunk 1 above)
Comment (jamietanna) [inline py/color_detect.py:62]: Sorry, ignore that, I forgot to remove that before the commit - just some testing work I was doing.

**Comment by jamietanna** [comment]
Sure thing. I've currently got some more changes that I'll be working on over the next few days, a main one being support for the i3 Window Manager (which I use). However, could possibly add support for more if needed.

**Pull request merged**